src/test/java/com/googlecode/jsonrpc4j/server/JsonRpcServerTest.java

```diff
@@ -16,241 +18,347 @@
-		Assert.assertEquals("gzip", response.getHeader(CONTENT_ENCODING));
-	}
-	
-	private String getCompressedResponseContent(byte[] compressed) throws IOException {
-		GZIPInputStream gzipInputStream = new GZIPInputStream(new ByteArrayInputStream(compressed));
-		InputStreamReader inputStreamReader = new InputStreamReader(gzipInputStream, StandardCharsets.UTF_8);
-		BufferedReader bufferedReader = new BufferedReader(inputStreamReader, 2048);
-		StringBuilder sb = new StringBuilder();
-		String readed;
-		while ((readed = bufferedReader.readLine()) != null) {
-			sb.append(readed);
-		}
-		return sb.toString();
-	}
-	
-	@Test
-	public void testGzipRequest() throws IOException {
-		MockHttpServletRequest request = new MockHttpServletRequest("POST", "/test-post");
-		request.addHeader(CONTENT_ENCODING, "gzip");
-		request.setContentType("application/json");
-		byte[] bytes = "{\"jsonrpc\":\"2.0\",\"id\":123,\"method\":\"testMethod\",\"params\":[\"Whir?inaki\"]}".getBytes(StandardCharsets.UTF_8);
-		
-		ByteArrayOutputStream baos = new ByteArrayOutputStream();
-		GZIPOutputStream gos = new GZIPOutputStream(baos);
-		gos.write(bytes);
-		gos.close();
-		
-		request.setContent(baos.toByteArray());
-		
-		MockHttpServletResponse response = new MockHttpServletResponse();
-		jsonRpcServer = new JsonRpcServer(Util.mapper, mockService, ServiceInterface.class, true);
-		jsonRpcServer.handle(request, response);
-		
-		String responseContent = new String(response.getContentAsByteArray(), StandardCharsets.UTF_8);
-		Assert.assertEquals(responseContent, "{\"jsonrpc\":\"2.0\",\"id\":123,\"result\":null}\n");
-		Assert.assertNull(response.getHeader(CONTENT_ENCODING));
-	}
-	
-	@Test
-	public void testGzipRequestAndResponse() throws IOException {
-		MockHttpServletRequest request = new MockHttpServletRequest("POST", "/test-post");
-		request.addHeader(CONTENT_ENCODING, "gzip");
-		request.addHeader(ACCEPT_ENCODING, "gzip");
-		request.setContentType("application/json");
-		byte[] bytes = "{\"jsonrpc\":\"2.0\",\"id\":123,\"method\":\"testMethod\",\"params\":[\"Whir?inaki\"]}".getBytes(StandardCharsets.UTF_8);
-		
-		ByteArrayOutputStream baos = new ByteArrayOutputStream();
-		GZIPOutputStream gos = new GZIPOutputStream(baos);
-		gos.write(bytes);
-		gos.close();
-		
-		request.setContent(baos.toByteArray());
-		
-		MockHttpServletResponse response = new MockHttpServletResponse();
-		jsonRpcServer = new JsonRpcServer(Util.mapper, mockService, ServiceInterface.class, true);
-		jsonRpcServer.handle(request, response);
-		
-		byte[] compressed = response.getContentAsByteArray();
-		String sb = getCompressedResponseContent(compressed);
-		
-		Assert.assertEquals(sb, "{\"jsonrpc\":\"2.0\",\"id\":123,\"result\":null}");
-		Assert.assertEquals("gzip", response.getHeader(CONTENT_ENCODING));
-	}
-	
-	// Service and service interfaces used in test
-	
-	public interface ServiceInterface {
-		String testMethod(String param1);
-	}
+
+    @Mock(type = MockType.NICE)
```

Comment:
I know that the indentation was incorrect in this file, however would you mind changing it back to the way it was before? It's tough to track exactly what changed when so many unrelated lines change in a diff.